scripts/metric_reporter/reporter/base_reporter.py

```diff
@@ -79,3 +78,16 @@ def output_csv(self, report_path: Path) -> None:
             error_msg: str = next(m for t, m in error_mapping.items() if isinstance(error, t))
             self.logger.error(error_msg, exc_info=error)
             raise ReporterError(error_msg) from error
+
+    def update_table(self, client, project_id: str, dataset_name: str) -> None:
+        """Update the BigQuery table.
+
+        Args:
+            client (Client): The client to interact with BigQuery.
+            project_id (str): The BigQuery project ID.
+            dataset_name (str): The BigQuery dataset name.
+
+        Raises:
+            NotImplementedError: If the method is not implemented by a subclass.
+        """
+        raise NotImplementedError("Subclasses must implement the `update_bigtable` method.")
```

Comment:
Should this be `update_bigtable` or `update_table`?
```sh
git grep -Ei "[a-z_]+\.update_bigtable\(" | wc -l # 0
git grep -Ei "[a-z_]+\.update_table\("    | wc -l # 15
```

Reply:
see https://github.com/mozilla/ecosystem-test-scripts/pull/37/commits/966d00f64055015ca13507fa181b1a847fe145d8